### ISSUE TITLE
White boxes in Evil Dead: Hail to the King 

### ISSUE THREAD

**Issue by AbandonedWitchStarlight** [flyinghead/flycast#238]
**Please Note: This form is the minimum required information for submitting bugs.**  
**Removing this form may lead to your issue being closed until it is completed.**

Platform / OS / Hardware: Windows 7 x64

Hardware: i3-3210, 4GB RAM, GT 620

**_Description of the Issue_**

Apparently Evil Dead Hail to the King has problems in Flycast, because there are white boxes that fill parts of the screen, I tried OpenGL and Directx (I'm using a build that has DirectX) but still the problem persists.

**_Debugging Steps Tested_**

 Switched to either OpenGl or DirectX, the problem still exists.

**_Logs Gathered_**

```

Please paste the contents of the log / logs here
You may leave this field blank if you have none.

```

**_Screenshots_**

![eddc-whitescreen](https://user-images.githubusercontent.com/53141496/118222066-73350e00-b4b1-11eb-8011-438455c25388.PNG)

**Comment by flyinghead** [comment]
This is a known issue with this game and is due to the use of a specific Dreamcast GPU feature (Secondary Accumulation Buffer). This feature is only emulated by the per-pixel renderers (Open GL and Vulkan).

**Comment by overlordchaos** [comment]
i guess this same issue applies to sonic adventure 2, that all the characters shadows looks like a big circle filled with black grid bars

**Comment by flyinghead** [comment]
No, Sonic Adventure 2 doesn't use this feature.

**Comment by overlordchaos** [comment]
> No, Sonic Adventure 2 doesn't use this feature.

then i open a new issue?

**Comment by flyinghead** [comment]
Yes. Make sure to indicate your platform (OS, GPU, etc.) as it might be platform-specific.
